depext.ml

```diff
@@ -41,11 +41,11 @@ let string_split char str =
   aux 0
 
 let has_command c =
-  let cmd = Printf.sprintf "/bin/sh -c command -v %s" c in
+  let cmd = Printf.sprintf "command -v %s" c in
```

Comment:
> Would it be possible for cmdliner to expose the find_cmd logic so that we dont have to cut and paste it here?

I don't think `cmdliner` is the place to expose such things. I'd say either just c&p or use Bos which [exposes](http://erratique.ch/software/bos/doc/Bos.OS.Cmd.html#VALexists) this logic in its API (but that's another dependency).